src/components/SelectionList/Search/ActionCell.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Badge from '@components/Badge';
+import Button from '@components/Button';
+import * as Expensicons from '@components/Icon/Expensicons';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+
+type ActionCellProps = {
+    onButtonPress: () => void;
+    action?: string;
+    isLargeScreenWidth?: boolean;
+};
+
+function ActionCell({onButtonPress, action = CONST.SEARCH.ACTION_TYPES.VIEW, isLargeScreenWidth = true}: ActionCellProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const theme = useTheme();
+    const StyleUtils = useStyleUtils();
+
+    if (action === CONST.SEARCH.ACTION_TYPES.PAID || action === CONST.SEARCH.ACTION_TYPES.DONE) {
+        const buttonTextKey = action === CONST.SEARCH.ACTION_TYPES.PAID ? 'iou.settledExpensify' : 'common.done';
+        return (
+            <Badge
+                text={translate(buttonTextKey)}
+                icon={Expensicons.Checkmark}
+                badgeStyles={[
+                    styles.ml0,
+                    styles.ph2,
+                    styles.gap1,
+                    isLargeScreenWidth ? styles.alignSelfCenter : styles.alignSelfEnd,
+                    StyleUtils.getBorderColorStyle(theme.border),
+                    StyleUtils.getHeight(variables.h20),
+                    StyleUtils.getMinimumHeight(variables.h20),
```

Comment:
```suggestion
                    StyleUtils.getHeight(variables.h20),
```
The minheight is redundant here

Reply:
I had to set it because we add a min height of 28 in [defaultBadge](https://github.com/Expensify/App/blob/54492997009d2b3703c0e41582af8d03ff7080c8/src/components/Badge.tsx#L71), so it was making the line height higher than it should be